### PR TITLE
[TAS-1523] ✨Add Airtable record for each new NFT Book/Collection

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/util/airtable.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      fields['Owner Name'] = ownerData.displayName;
+    }
+
+    await base('Publications').create([{ fields }]);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error(err);
+  }
+
+  return Promise.resolve();
```
Comment (williamchong) [inline src/util/airtable.ts]: We don't need to return anything if not needed

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/util/airtable.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  id: string;
+  ownerWallet: string;
+  type: string;
+  minPrice: number;
+  maxPrice: number;
+  imageURL: string;
+  iscnIdPrefix?: string;
+  iscnVersionAtMint?: number;
+}): Promise<void> {
+  if (TEST_MODE) return Promise.resolve();
```
Comment (williamchong) [inline src/util/airtable.ts]:
```suggestion
  if (TEST_MODE || ! base) return;
```

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/util/airtable.ts (hunk 2 above)
Comment (nwingt) [reply src/util/airtable.ts]: How about checking `!base` only?

**Review by williamchong — APPROVED** [review]

File: src/util/airtable.ts (hunk 2 above)
Comment (williamchong) [reply src/util/airtable.ts]: Sure

**Review by williamchong — APPROVED** [review]

**Pull request merged**